src/org/usfirst/frc4904/robot/subsystems/Arm.java

```diff
@@ -0,0 +1,40 @@
+package org.usfirst.frc4904.robot.subsystems;
+
+
+import org.usfirst.frc4904.robot.RobotMap;
+import org.usfirst.frc4904.standard.commands.Idle;
+import org.usfirst.frc4904.standard.custom.sensors.CANTalonEncoder;
+import org.usfirst.frc4904.standard.subsystems.motor.Motor;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+public class Arm extends Subsystem {
```

Comment:
I think you should name the class BoxIO and name the instance boxIO, as this isn't any normal arm. that's just my opinion though